_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -53596,6 +53699,23 @@
 	},
 /turf/simulated/floor/engine,
 /area/station/engineering/control)
+"kna" = (
+/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/machinery/atmospherics/pipe/simple/hidden/supply,
+/obj/machinery/power/apc{
```

Comment:
Replace these with the `/obj/machinery/power/apc/directional` subtypes

Reply:
fixed